project/frontend/src/containers/Explanation/Explanation.js

```diff
@@ -1,45 +1,60 @@
         <p>
-          Bacon ipsum dolor amet buffalo kielbasa turkey strip steak rump.
-          T-bone ribeye pork belly ham hock swine, cupim sirloin boudin. Chislic
-          turducken t-bone, ground round drumstick ham strip steak kielbasa.
-          Spare ribs tenderloin chislic porchetta landjaeger. Frankfurter beef
-          landjaeger t-bone kevin kielbasa. Biltong pork chop buffalo bacon.
-        </p>
-        <p>
-          Frankfurter pastrami doner flank short ribs bacon cow sausage pancetta
-          kielbasa t-bone fatback boudin. Rump sirloin pork chop jowl ribeye.
-          Hamburger brisket capicola chicken beef ham hock pork chop jerky
-          ribeye rump tail pig. Pastrami venison pork tongue, burgdoggen
-          capicola t-bone chuck salami ribeye corned beef pig drumstick jowl.
-          Flank buffalo ham hock doner filet mignon sausage. Doner tongue
-          pancetta flank. Shank pork beef ribs buffalo, rump jowl salami brisket
-          spare ribs pork loin short ribs ham hock.
-        </p>
-        <p>
-          Ham hock porchetta pork, meatloaf ball tip bacon buffalo. Salami
-          fatback rump venison porchetta, ham hock tenderloin chuck strip steak
-          turducken short ribs turkey chicken pork chop. Biltong pork loin chuck
-          meatball, bresaola brisket pork belly short loin turkey alcatra kevin
-          doner drumstick corned beef short ribs. Tenderloin kielbasa shoulder
-          hamburger buffalo strip steak. Sausage shank hamburger t-bone pork
-          belly pork jerky porchetta jowl.
+          SGonks work along similar ideas to virtual trading platforms, but
+          instead of investing in stocks, the player can invest “Trend Bucks” in
+          a Google search query. By buying into a search at a given point in
+          time, they can cash out later and earn or lose “Trend Bucks" based on
+          increased or decreased global interest in that query, as indicated by
+          Google trends data. This provides an interactive way for teams to
+          compete with one another to predict search trends overtime.
         </p>
       </div>
-      <Link to="/" className={classes.BackButton}>Back</Link>
+
+      <div className={classes.ExplanationContainer}>
+        <h1>How does it work?</h1>
+        <ol>
+          <li>
+            You can find all the competition you’re in in the or create a
+            competition if you don’t have any.
+          </li>
+          <li>
+            The aim of the game is to invest in the Google Search that you think
+            it’s popularity is going to increase in the future and end up with
+            the most trend bucks at the end of the competition.
+          </li>
+          <li>
+            You will receive trends bucks based on the increased percentage of
+            your google search investments. Same thing applies to losing trend
+            bucks when the popularity of your investment decreases.
```

Comment:
"You receive and lose Trend Bucks based on how many you invested in the query, and the proportional change in popularity of the query since you invested. For instance, if you invest 100 Trend Bucks in a query that doubles in popularity by the time you sell, you cash out with 200 Trend Bucks, and 100 Trend Buck profit."